src/lib/common/limits.h

```diff
@@ -65,6 +65,8 @@
 #define IP_LENGTH_MAX           15    // Based on xxx.xxx.xxx.xxx
 #define STRING_SIZE_FOR_INT     16    // Room enough for an integer
 #define STRING_SIZE_FOR_DOUBLE  64    // Room enough for a double
+#define CORRELATOR_ID_SIZE      36
```

Comment:
I'd suggest to add a little comment about where that 36 comes, as in the cases above, e.g:

"Max size of a UUIDv4 string"

or something like that.